File: backend/components/metiers/voyage/utils/logger/helpers/logger_helper.py
# logger_helper.py
# Helper logger Python pour Voyage – exemple clé en main
from datetime import datetime


def format_log(level, message):
    """
    Formate un message de log avec niveau et timestamp
    """
    return f"[{level.upper()}][{datetime.utcnow().isoformat()}] {message}"

File: backend/components/metiers/mode/utils/logger/helpers/logger_helper.js
// logger_helper.js
// Helper logger JS pour Mode – exemple clé en main

/**
 * Formate un message de log avec niveau et timestamp
 * @param {string} level
 * @param {string} message
 * @returns {string}
 */
function formatLog(level, message) {
  return `[${level.toUpperCase()}][${new Date().toISOString()}] ${message}`;
}

module.exports = { formatLog };
